server/services/lists/approverCaseloadService.test.ts

```diff
@@ -1,124 +1,56 @@
 import LicenceApiClient from '../../data/licenceApiClient'
 import type { User } from '../../@types/CvlUserDetails'
-import type { OffenderDetail } from '../../@types/probationSearchApiClientTypes'
-import LicenceStatus from '../../enumeration/licenceStatus'
-import LicenceType from '../../enumeration/licenceType'
 import ApproverCaseloadService from './approverCaseloadService'
-import type {
-  CaseloadItem,
-  CvlFields,
-  CvlPrisoner,
-  LicenceSummaryApproverView,
-} from '../../@types/licenceApiClientTypes'
+import type { ApprovalCase } from '../../@types/licenceApiClientTypes'
 import { parseIsoDate } from '../../utils/utils'
 
 jest.mock('../communityService')
 jest.mock('../licenceService')
 jest.mock('../../data/licenceApiClient')
 
-const communityOffender = {
-  otherIds: { nomsNumber: 'AB1234E', crn: 'X12348' },
-  offenderManagers: [{ active: true, staff: { forenames: 'Joe', surname: 'Bloggs', code: 'X1234' } }],
-} as OffenderDetail
-
 describe('Approval Caseload Service', () => {
-  const communityService = new CommunityService(null, null) as jest.Mocked<CommunityService>
   const licenceApiClient = new LicenceApiClient(null) as jest.Mocked<LicenceApiClient>
 
-  const serviceUnderTest = new ApproverCaseloadService(communityService, licenceApiClient)
+  const serviceUnderTest = new ApproverCaseloadService(licenceApiClient)
   const user = {
     deliusStaffIdentifier: 2000,
     probationTeamCodes: ['teamA', 'teamB'],
     prisonCaseload: ['p1', 'p2'],
   } as User
 
-  const caseloadItem = {
-    prisoner: {
-      prisonerNumber: 'AB1234E',
-      conditionalReleaseDate: '2024-05-25',
-      firstName: 'THE',
-      lastName: 'PRISONER',
-      status: 'ACTIVE IN',
-    } as CvlPrisoner,
-    cvl: {} as CvlFields,
-  } as CaseloadItem
-
-  const staffDetails = {
-    username: 'joebloggs',
-    staffCode: 'X1234',
-    staff: {
-      forenames: 'Joe',
-      surname: 'Bloggs',
-    },
-  }
-
   afterEach(() => {
     jest.resetAllMocks()
   })
 
   describe('approval needed', () => {
-    const licence = {
-      kind: 'CRD',
-      nomisId: 'AB1234E',
+    const approvalCase = {
       licenceId: 1,
-      licenceType: LicenceType.AP,
-      licenceStatus: LicenceStatus.SUBMITTED,
+      approvedBy: 'Jim Smith',
+      approvedOn: '25 Apr 2014',
+      isDueForEarlyRelease: false,
+      name: 'The Prisoner',
+      prisonerNumber: 'AB1234E',
+      releaseDate: '25 May 2024',
+      sortDate: parseIsoDate('2024-05-25'),
       submittedByFullName: 'An Submitter',
-      comUsername: 'joebloggs',
-      isReviewNeeded: false,
-      isInHardStopPeriod: false,
-      isDueForEarlyRelease: true,
-      isDueToBeReleasedInTheNextTwoWorkingDays: false,
-    } as LicenceSummaryApproverView
+      urgentApproval: false,
+      probationPractitioner: {
+        staffCode: 'X1234',
+        name: 'Joe Bloggs',
+      },
+    } as ApprovalCase
 
     it('builds the approval needed caseload', async () => {
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**thought:** ‏- don't know if these tests now have any value as we are just testing whether the mock is returned rather than any other functionality. Get rid?

Reply:
**suggestion(non-blocking):**  that would make sense - think we should remove these as part of the overall story